met-web/src/components/engagement/view/EmailPanel.tsx

```diff
@@ -0,0 +1,128 @@
+                </Typography>
+            </Grid>
+            <Grid item xs={12}>
+                <Typography>You will receive a link to access the survey at the email address you provide.</Typography>
+            </Grid>
+
+            <Grid item xs={12}>
+                <Typography sx={{ mb: 2, fontWeight: 'bold' }}>Email address</Typography>
+                <TextField
+                    onChange={(e) => props.updateEmail(e.target.value)}
+                    id="outlined-basic"
+                    label="Outlined"
+                    variant="outlined"
+                    error={emailError}
+                    helperText={'Please Enter an Email'}
+                />
+            </Grid>
+            <Grid
+                item
+                container
+                direction="row"
+                width="100%"
+                xs={12}
+                alignItems="center"
+                justifyContent="flex-start"
+                rowSpacing={1}
+            >
+                <FormControl required error={termsError} component="fieldset" sx={{ m: 1 }} variant="standard">
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                onChange={(event) => {
+                                    setChecked(event.target.checked);
+                                }}
+                                {...label}
+                                name="gilad"
+                            />
+                        }
+                        label={<Typography>I agree to the Terms and Conditions below.</Typography>}
+                    />
+                    {termsError ? <FormHelperText>Please Accept the Terms and Conditions</FormHelperText> : <></>}
+                </FormControl>
+            </Grid>
+            <Grid item xs={12} sx={{ pt: 1, pb: 1, borderLeft: 8, borderColor: '#003366' }}>
+                <Typography paragraph={true} id="modal-modal-description" sx={{ pl: 1 }}>
+                    "On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and
+                    demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee
+                    the pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their
+                    duty through weakness of will, which is the same as saying through shrinking from toil and pain.
+                </Typography>
+            </Grid>
+            <Grid item container direction="row" justifyContent="flex-end" alignItems="flex-end" xs={12} rowSpacing={1}>
+                <Button variant="outlined" onClick={props.handleClose} sx={{ m: 1 }}>
+                    Cancel
+                </Button>
+                <Button
+                    onClick={
+                        !termsError && !emailError
+                            ? () => props.checkEmail()
```

Comment:
I would encourage following the similar onClick pattern in EngagementForm. clicks -> validates form -> if error setErrors and components will becomered, if success continue

Reply:
good idea